apps/events-helsinki/src/pages/articles/index.tsx

```diff
@@ -214,12 +213,21 @@ export async function getStaticProps(context: GetStaticPropsContext) {
         language: getQlLanguage(language).toLocaleLowerCase(),
       },
     });
-
+    if (!pageData) {
+      return {
+        notFound: true,
+        revalidate: true,
+      };
+    }
     const page = pageData.pageByTemplate;
     return {
       props: {
         page,
-        ...(await serverSideTranslationsWithCommon(language)),
+        ...(await serverSideTranslationsWithCommon(language, [
+          'common',
+          'home',
+          'cms',
+        ])),
```

Comment:
instead, the `'cms'` could be added to the `serverSideTranslationsWithCommon`.  I actually think that it was already done, but something has went wrong while rebasing. At leat the `'common'` and `'home'`  are not needed, because they are allready listed in the 
```
const COMMON_TRANSLATIONS = [
  'appHobbies',
  'common',
  'consent',
  'errors',
  'footer',
  'notFound',
  'home',
];
```